cincinnati/src/plugins/internal/arch_filter.rs

```diff
@@ -51,6 +51,46 @@ impl ArchFilterPlugin {
 
         Ok(Box::new(plugin))
     }
+
+    pub fn remove_conditional_edges(
+        &self,
+        ce: &mut Vec<cincinnati::ConditionalEdge>,
+        arch: String,
+    ) -> Fallible<()> {
+        ce.into_iter()
+            .for_each(|ce: &mut cincinnati::ConditionalEdge| {
+                ce.edges
+                    .retain(|e| e.from.contains(&arch) || e.to.contains(&arch));
+            });
+
+        Ok(())
+    }
+
+    pub fn rm_ce_arch_identifier(
```

Comment:
nit: I'd rather replace `ce` with `conditional_edge` here, because we'll presumably have more folks reading the code than writing it, and had a brief "what's `ce`?" moment, before I got to the function's argument list.

Reply:
renamed